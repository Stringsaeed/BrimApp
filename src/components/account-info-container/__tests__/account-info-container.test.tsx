import { screen } from "@testing-library/react-native";
import React from "react";
import { Text } from "react-native";

import AccountInfoContainer from "../";
import render from "../../../utils/test";

describe("AccountInfoContainer", () => {
  it("should render correctly", () => {
    render(
      <AccountInfoContainer>
        <Text>Test</Text>
      </AccountInfoContainer>
    );
    expect(screen.toJSON()).toMatchSnapshot();
  });

  it("should render children", () => {
    render(
      <AccountInfoContainer>
        <Text>Test</Text>
      </AccountInfoContainer>
    );
    expect(screen.getByText("Test")).toBeTruthy();
  });

  it("should render with default styles", () => {
    render(
      <AccountInfoContainer>
        <Text>Test</Text>
      </AccountInfoContainer>
    );

    expect(screen.getByTestId("account-info-container")).toHaveStyle({
      paddingRight: 18,
      paddingLeft: 18,
      paddingTop: 24,
      flex: 1,
      gap: 24,
    });
  });
});
