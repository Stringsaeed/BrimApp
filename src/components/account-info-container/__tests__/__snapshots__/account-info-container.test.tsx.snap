// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountInfoContainer should render correctly 1`] = `
<View
  animatedStyle={
    {
      "value": {},
    }
  }
  collapsable={false}
  layout={[Function]}
  style={
    {
      "flex": 1,
      "gap": 24,
      "paddingLeft": 18,
      "paddingRight": 18,
      "paddingTop": 24,
    }
  }
  testID="account-info-container"
>
  <Text>
    Test
  </Text>
</View>
`;
